code/lib/cli/package.json

```diff
@@ -81,6 +81,7 @@
     "globby": "^11.0.2",
     "jscodeshift": "^0.14.0",
     "leven": "^3.1.0",
+    "prettier": "^2.8.4",
```

Comment:
@kasperpeulen Please make sure all the prettier version specifiers match across the monorepo

Reply:
Done